.evergreen.yml

```diff
@@ -41,6 +41,30 @@ functions:
           GOROOT: ${go_root}
           <<: *go_options
         command: make test
+  "atlas e2e":
+    - command: subprocess.exec
+        type: test
+        params:
+          add_to_path:
+            - ${go_bin}
+          working_dir: src/github.com/mongodb/mongocli/e2e/atlas
```

Comment:
if your working dir is `src/github.com/mongodb/mongocli/e2e/atlas` you won't have access to the `male file` which is a couple of levels before

Reply:
I have changed all the e2e file to reflect that change. I have also tested that those two new commands are working as expected

Reply:
I thought you meant that the make e2e-test-atlas would not work when running` make e2e-test-atlas` inside the terminal (that is what I tested). Here you mean that the evergreen task cannot access the make file I guess. I suppose there is a way out.

Reply:
I am going to try `../../make e2e-test-atlas`

Reply:
the correct solution here is to change your working dir to be `src/github.com/mongodb/mongocli`

Reply:
mh... The reason why I did that change is that I thought that working_dir must contain the directory of the e2e test (I don't know why, I guess because I associated working directory to "where the job must be done" instead of "where is the file that run my test". I am going to update working_dir with src/github.com/mongodb/mongocli

Reply:
`working dir` is where you need to run your commands evergreen has not context on what your test are or how they work, since your plan is to run `make`, and you need a makefile for that then you should work from the folder where that makefile is, the make file already has the logic on how to run those test and you should not [leak](https://en.wikipedia.org/wiki/Leaky_abstraction) that up to evergreen